docs/manual/other-topics/typescript.md

```diff
@@ -46,28 +44,28 @@ interface UserCreationAttributes extends Optional<UserAttributes, "id"> {}
 
 class User extends Model<UserAttributes, UserCreationAttributes>
   implements UserAttributes {
-  public id!: number; // Note that the `null assertion` `!` is required in strict mode.
-  public name!: string;
-  public preferredName!: string | null; // for nullable fields
+  declare id: number; // Note that the `null assertion` `!` is required in strict mode.
```

Comment:
Remove `// Note that the `null assertion` `!` is required in strict mode.` comment.

Reply:
Oh yeah good catch. Do you want to open a follow-up PR?